pocketimdb/movie_feedbacks/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from pocketimdb.movies.models import Movie
+from pocketimdb.users.models import User
+
+class MovieFeedback(models.Model):
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    feedback = models.BooleanField()
```

Comment:
ovo kao sto vidis moze imati pogresno znacenje, jer recimo ako ti stoji false onda bi to znaci feedback -> false, ja bi pomislio kao da nema feedback-a, a ustavri kod tebe predstavlja dislike. Hajde ovo malo drugacije da odradimo, tipa da se ovaj feedback ne zove feedback model nego da bude like i onda za taj like imas ili neku enumeraciju ili mozes ici samo sa 1 i -1 jer prilikom racunanja ce ti biti mozda lakse samo da sumiras sve like-ove i dislike-ove nezavisno jedan od drugog. 

Takodje ovaj model nepa potrebe da bude vam movies modula, ovo je dio filmova i onda kao takav ide uz onaj modul. Ti mozes imati vise modela na nivou jednog modula, samim tim sto ti se file zove `model(s)` nam to i govori